// Copyright (c) Alex Jamieson-Binnie. All rights reserved.
// Licensed under the GPL. See License.txt in the project root for license information.

syntax = "proto3";

import "narupa/protocol/instance/get_topology.proto";
import "narupa/protocol/instance/get_frame.proto";

package narupa.protocol.instance;

service MoleculeProvider {
  rpc SubscribeTopology (GetTopologyRequest) returns (stream GetTopologyResponse);
  rpc SubscribeFrame (GetFrameRequest) returns (stream GetFrameResponse);
}
